apps/prairielearn/src/admin_queries/institutions_usage.json

```diff
@@ -0,0 +1,15 @@
+{
+  "description": "Institutions usage data.",
```

Comment:
This is the admin query that we'll run to figure out which institutions had any significant usage at all.